packages/web/src/components/gcds-side-nav/gcds-side-nav.css

```diff
@@ -1,34 +1,53 @@
-@layer reset, defaults;
+@layer reset, default, desktop, mobile;
 
 @layer reset {
-  :host * {
-    margin: 0;
-    box-sizing: border-box;
+  :host {
+    display: block;
+
+    * {
+      margin: 0;
+      box-sizing: border-box;
+    }
   }
 }
 
-@layer defaults {
+@layer default {
   :host {
-    display: block;
     width: 100%;
 
-    @media only screen and (width >= 64em) {
-      max-width: var(--gcds-side-nav-max-width);
-    }
-
     .gcds-side-nav__heading {
       font: var(--gcds-side-nav-heading-font);
       margin-block-end: var(--gcds-side-nav-heading-margin);
       padding: var(--gcds-side-nav-heading-padding);
+    }
+  }
+}
+
+/* Note: viewport specific style decision */
+@layer desktop {
+  :host .gcds-side-nav {
+    @media only screen and (width >= 64em) {
+      max-width: var(--gcds-side-nav-max-width);
+    }
+  }
+}
 
-      @media only screen and (width < 64em) {
+/* Note: viewport specific style decision */
+@layer mobile {
+  @media only screen and (width < 64em) {
+    :host {
+      .gcds-side-nav__heading {
         display: block;
         width: 0;
         height: 0;
         margin: 0;
         padding: 0;
         overflow: hidden;
       }
+
+      gcds-nav-group {
```

Comment:
I noticed this line of `padding: 0` on the `gcds-nav-group` is causing the close button in the mobile overlay to push right up against the edges of the container. When it receives focus the focus outline is cut off on either side making it look a little weird.